scm-ui/ui-components/src/Tooltip.tsx

```diff
@@ -38,7 +38,7 @@ class Tooltip extends React.Component<Props> {
 
   render() {
     const { className, message, location, children } = this.props;
-    const multiline = message.length > 60 ? "has-tooltip-multiline" : "";
+    const multiline = message.length > 60 ? "is-multiline" : "";
```

Comment:
I don't see a change with the `is-multiline` class. With or without this class multiline strings seem to work. I could not find any class named `is-multiline` nor `has-tooltip-multiline` in our styles.